src/Containers/Navigation.tsx

```diff
@@ -35,6 +35,7 @@ export interface NavigationProps extends MainInterface, ResponsiveInterface {
     tag?: React.ReactElement | string;
     theme: DefaultTheme;
     label?: string;
+    footer?: string | number | React.FC;
```

Comment:
Isn't this only React Element based on the code?

Reply:
Based on its current use case yes, but I just made it as flexible as possible so that anything could be passed as a footer if necessary. I can change it to just React.FC if you think that makes more sense?

Reply:
Yeah but does it actually support Text being passed in?

Reply:
Like If i pass in a number does the file have <Text>{children}</Text>
because if you support React.FC that will collide with the above ^

Reply:
hmm good catch, should be ```React.ReactElement```

Reply:
@nikitasheremet

Reply:
So leave string, and number and replace React.FC with ReactElement? Or have just ReactElement? @ralph-dev

Reply:
Just have react.reactelement